### DISCUSSION TITLE
error start auto dj

### DISCUSSION THREAD

**Discussion by djmafiaem** [AzuraCast/AzuraCast#6318] [General]
after I updated, it gives me an error when starting autodj. : AlreadyRunningException at /var/azuracast/www/src/Exception/SupervisorException.php L41: station_1:station_1_backend cannot start
It is already running.    
a solution?
Version : Rolling Release #[35a6c8c] (2023-06-05 12:18) • Docker • PHP 8.2

**Comment by flashradio** [comment]
Something similar happend to me

**Comment by flashradio** [comment]
When I upgrade from version 1.81 to 1.83 the station is in silence

**Comment by flashradio** [comment]
Appears as station off

**Comment by flashradio** [comment]
Now with this problem I have had to create another station, but is there a way to clone the playlists to the new one?

**Reply by Moonbase59** [reply]
The easiest would be to clone the complete station and make changes afterwards, I’d think. If using the same media, you could also export the playlists and re-import them in the other station. (Not feasible for me, I currently use 56 playlists in a station.) AzuraCast provides for both variants.

**Comment by flashradio** [comment]
Once we will try to clone but does not clone the music files

**Reply by Moonbase59** [reply]
That’s true. You could share the media location(s), effectively using the same files, or copy all your media (locally or—if on a remote server—using SFTP, for instance).

**Comment by flashradio** [comment]
Yes we are clone all the media files, but we have 71 playlist that does not clone. So I have to write all the playlists and add on it all the musical files in the new station, are 12.000 files. A lot of work all due to the failure of an update

**Comment by flashradio** [comment]
The issue persists even with new one just create station

**Comment by djmafiaem** [comment]
it seems that, with the new update, it recovered and works perfectly. Thanks for the fix

**Reply by flashradio** [reply]
There is a new update?

**Comment by flashradio** [comment]
It seems that every time you clone a station to another because of some problem, it drags the same problems to the new one.

**Comment by flashradio** [comment]
It need it a new update that solved the issues of the last one

**Comment by flashradio** [comment]
Yesterday, after importing the list to the new one and configuring the programming with its days and times, when it was restarted it remained silent and turned off, although in the scheduled programming it looked like it was working.

**Reply by djmafiaem** [reply]
did you finally fix the problems?

**Comment by flashradio** [comment]
No, not yet. Every time I try to do and scheduled the lists, the station go
silence. I am waiting to the next Azuracast update to see if that solved my
problem. Now the station is played but with any programming.
